parsec/interfaces/ptg/ptg-compiler/jdf2c.c

```diff
@@ -6642,8 +6642,6 @@ static void jdf_generate_code_hook_cuda(const jdf_t *jdf,
                 "#endif /* PARSEC_PROF_TRACE */\n");
     }
 
-    dyld = jdf_property_get_string(body->properties, "dyld", NULL);
```

Comment:
This change is unrelated (and incomplete if intentional, we still have a lot of infrastructure to do the 'dyld' thing, and DPLASMA is very dependent on it)

Reply:
it is intentional, the property was extracted twice without modification between them. So this call is unnecessary.